src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/WmsGetFeatureInfoTest.java

```diff
@@ -72,6 +73,7 @@ public void testGetFeatureInfoGML() throws Exception {
 
     @Test
     public void testGetFeatureInfoGMLReprojection() throws Exception {
+        if (isGeopkgTest()) return;
```

Comment:
Why?

Reply:
It has been disable due the different axis order used for geometries in Geopackage comparing to the one used in postgres tests db.